src/Ui/RedirectToOrderShowAction.php

```diff
@@ -36,6 +36,7 @@ public function __construct(RouterInterface $router, OrderRepositoryInterface $o
     public function __invoke(Request $request): Response
     {
         $number = $request->attributes->get('number');
+        Assert::string($number);
```

Comment:
While you're at it, change the var name to `$orderNumber`.
Although not strictly related, a construct like `Assert::string($number)` looks icky.